catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -65,6 +65,16 @@ public void testSingleSchematron()
         getService("dog_legs.sch").validate(getMetacard("dog_4leg_3paw.xml"));
     }
 
+    @Test
+    public void testNullorEmptyMetadata()
+            throws ValidationException, IOException, SchematronInitializationException {
+        MetacardImpl card = new MetacardImpl();
+        SchematronValidationService service = getService("dog_legs.sch");
+        service.validate(card);
+        card.setMetadata("");
+        service.validate(card);
+    }
```

Comment:
Junit is lacking in this regard and doesn't provide an analog to Spock's `notthrown`. Could you add a comment here indicating that's what you're implicitly asserting? Otherwise, a future dev might see this and mistakenly think it's a test with no assertions and "clean it up" by removing it.